var/spack/repos/builtin/packages/r-rgraphviz/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RRgraphviz(RPackage):
```

Comment:
`Rgraphviz`?

Reply:
The package name is Rgraphviz, and it's r package, so add an "R" before the package name.

Reply:
but `Rgraphviz` i think already indicates that it's `R` package of `graphviz`, so one `R` should be enough, isn't it?

Reply:
No, I tried to delete one R, it reported error:  Error: 'module' object has no attribute 'RRgraphviz'. So I think two Rs are needed. Some other R packages starting with r also have two Rs.

Reply:
ok, just looked strange